### ISSUE TITLE
Error	C1083	Cannot open include file: 'pcl/filters/voxel_grid.h': No such file or directory

### ISSUE THREAD

**Issue by monajalal** [PointCloudLibrary/pcl#1938]
I had an OpenARK-SDK project that could detect the PCL libarary but then I made a OpenARK-test project in the same solution side by side which I use to test the OpenARK-SDK. I have also set OpenARK-SDK as reference for OpenARK-test.
![pcl](https://user-images.githubusercontent.com/1892917/28232531-0c42b658-68a6-11e7-9ace-5f6dda0be5de.PNG)

```
Severity	Code	Description	Project	File	Line	Category	Source	Suppression State	Tool
Error	C1083	Cannot open include file: 'pcl/filters/voxel_grid.h': No such file or directory	OpenARK-test	c:\OpenARK\Plane.h	7	fatal	Build		

```

Here's my linker additional inputs for OpenARK-SDK: https://pastebin.com/y1sievuj
Here's Linker additional library dependencies for OpenARK-SDK: https://pastebin.com/T2EYT13B
Here's additional C++ libraries for OpenARK-SDK: https://pastebin.com/pWUgKS1V

Because I set OpenARK-SDK as a reference for OpenARK-test, I am not setting these for OpenARK-test.

How can I solve this error?
Basically Plane.h is the first one it hits for libaries and the first library is:
```
// PCL Libaries
#include <pcl/filters/voxel_grid.h>
```

**Comment by SergioRAgostinho** [comment]
Please use cmake to use pcl in your project. Other methods are not officially supported.
